src/groovy/au/org/emii/aatams/detection/DetectionView.groovy

```diff
@@ -85,7 +85,7 @@ class DetectionView extends Detection implements Embargoable {
 
     def applyEmbargo(allowSanitised = true) {
 
-        def releaseEmbargoed = surgery.release.isEmbargoed()
+        def releaseEmbargoed = surgery?.release?.isEmbargoed()
```

Comment:
Do you think it'd be better to rename this method and refactor to something like:

```groovy
def applySecurityRestrictions(allowSanitised = true) {
    applyEmbargo(...)
    applyProjectProtection(...)
}
```

Seems as though there's quite a bit going on in here now.
It might be a biggish change (the method rename mainly) so I'm not too hung up on doing a refactor as part of this PR.